arrays/array_practice.rb

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env ruby
+
+#Create an array variable with the the numbers 1 through 10
+array = (1..10).to_a
+# 1...2...3...4...5...6...7...8...9...10...
+puts "#{array.join('...')}" + "..."
```

Comment:
`puts array.join('...') + '...'`

No need for the interpolation if you have nothing else in the string...